### PR TITLE
Command exercises 204641

### PR DESCRIPTION
**Pull request by MeringueMayar**

### CODE REVIEW

File: src/main/java/edu/iis/powp/command/ComplexCommand.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    
+    private int i, i1;
+    private ArrayList<IPlotterCommand> commandList = new ArrayList<IPlotterCommand>();
+
+    public ComplexCommand() {
+        this.i = i;
+        this.i1 = i1; 
+    }
+    
+    public void addCommand(IPlotter plotter, IPlotterCommand command){
```
Comment (kowallus) [inline src/main/java/edu/iis/powp/command/ComplexCommand.java:26]: plotter argument should be removed.

File: src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java [hunk 2]
```diff
@@ -5,10 +5,22 @@
...
 
+import edu.iis.client.plottermagic.IPlotter;
 /**
  *
  * @author karko
  */
-public class CommandDrawLineToPosition {
+public class CommandDrawLineToPosition implements IPlotterCommand{
+
+    private int i, i1;
```
Comment (kowallus) [inline src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java]: Unmeaningful names of attributes

File: src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java [hunk 3]
```diff
@@ -5,10 +5,22 @@
...
     
+    public CommandDrawLineToPosition(int i, int i1) {
```
Comment (kowallus) [inline src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java]: Unmeaningful names of parameters.

File: src/main/java/edu/iis/powp/command/CommandFactory.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    
+    public static IPlotterCommand drawSquare(){
+        ComplexCommand complexCommand = new ComplexCommand();
+        
+        CommandDrawLineToPosition commandDrawLineToPosition = new CommandDrawLineToPosition(20,0);
+        CommandDrawLineToPosition commandDrawLineToPosition1 = new CommandDrawLineToPosition(20,20);
+        CommandDrawLineToPosition commandDrawLineToPosition2 = new CommandDrawLineToPosition(0,20);
+        CommandDrawLineToPosition commandDrawLineToPosition3 = new CommandDrawLineToPosition(0,0);
+        
+        complexCommand.addCommand(Application.getComponent(DriverManager.class).getCurrentPlotter(),
```
Comment (kowallus) [inline src/main/java/edu/iis/powp/command/CommandFactory.java:25]: Coupling with Application:
1. could be easily avoided (parameter),
2. actually unnecessary.
Application.getComponent(DriverManager.class).getCurrentPlotter() duplicated in code many times!

**Review by kowallus — COMMENTED** [review]
4 pts

File: src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java (hunk 2 above)
Comment (MeringueMayar) [reply src/main/java/edu/iis/powp/command/CommandDrawLineToPosition.java]: Just using already existing names! Better to be consistent with the available code.

**Pull request closed without merging**